### PR TITLE
doc: update doc versions for 2.2.0 release

### PR DESCRIPTION
**Pull request by peknis**
Update doc versions  for
2.2.0 nRF Connect SDK release
NCSDK-18303

Signed-off-by: Pekka Niskanen <pekka.niskanen@nordicsemi.no>

- [x] Confirm versions for nrfx,tfm,mcuboot
- [x]  Add manifest update for nrfxlib version bump

### CODE REVIEW

**Comment by peknis** [comment]
@osaether, please confirm the versions for nrfx and tfm.

File: doc/versions.json [hunk 1]
```diff
@@ -35,8 +36,17 @@
   ],
   "COMPONENTS_BY_VERSION": {
     "latest": {
-      "ncs": "2.1.99",
-      "nrfxlib": "2.1.99",
+      "ncs": "2.2.0",
+      "nrfxlib": "2.2.0",
+      "zephyr": "3.2.99",
+      "mcuboot": "1.9.99",
+      "nrfx": "2.9",
```
Comment (osaether) [inline doc/versions.json:43]: correct nrfx versions is 2.10

File: doc/versions.json [hunk 2]
```diff
@@ -35,8 +36,17 @@
...
+      "tfm": "1.6.0",
+      "matter": ""
+    },
+    "2.2.0": {
+      "ncs": "2.2.0",
+      "nrfxlib": "2.2.0",
       "zephyr": "3.2.99",
       "mcuboot": "1.9.99",
       "nrfx": "2.9",
```
Comment (osaether) [inline doc/versions.json:52]: correct nrfx versions is 2.10

File: doc/versions.json (hunk 1 above)
Comment (umapraseeda) [reply doc/versions.json]: updated

File: doc/versions.json (hunk 2 above)
Comment (umapraseeda) [reply doc/versions.json]: updated

**Review by greg-fer — APPROVED** [review]

**Review by gmarull — APPROVED** [review]

**Review by osaether — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]

**Review by shantha-14 — APPROVED** [review]

**Pull request merged**